keyboards/kbdfans/kbd67/rev2/rules.mk

```
@@ -32,4 +32,4 @@ AUDIO_ENABLE = no           # Audio output on port C6
 FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
 HD44780_ENABLE = no 		# Enable support for HD44780 based LCDs
 
-LAYOUTS = 65_ansi 65_iso
+LAYOUTS = 65_ansi 65_iso 65_ansi_blocker 65_ansi_blocker_splitbs
```

`65_ansi_blocker_splitbs` isnt an existing community layout
```suggestion
LAYOUTS = 65_ansi 65_iso 65_ansi_blocker
```